tests/panels/charts/xy/test_xy.py

```
@@ -333,3 +333,115 @@ async def test_area_percentage_chart() -> None:
+                'aggregation': 'average',
+                'field': 'error_rate',
+                'id': 'metric2',
+                'axis': 'right',
+                'color': '#FF5252',
+                'line_width': 3,
+                'line_style': 'dashed',
+            },
+        ],
+    }
+    lens_chart = LensLineChart.model_validate(lens_config)
+    layer_id, kbn_columns, kbn_state_visualization = compile_lens_xy_chart(lens_xy_chart=lens_chart)
```

Ruff (0.14.10)

Unpacked variable `layer_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)